specification/dataprotection/resource-manager/Microsoft.DataProtection/stable/2021-07-01/examples/ResourceGuardCRUD/GetDefaultOperationsRequestObject.json

```diff
@@ -0,0 +1,18 @@
+{
+  "parameters": {
+    "subscriptionId": "0b352192-dcac-4cc7-992e-a96190ccc68c",
+    "resourceGroupName": "SampleResourceGroup",
+    "resourceGuardsName": "swaggerExample",
+    "requestName": "default",
+    "api-version": "2021-07-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "subscriotions/0b352192-dcac-4cc7-992e-a96190ccc68c/resourceGroups/SampleResourceGroup/providers/Microsoft.DataProtection/resourceGuards/swaggerExample/disableSoftDeleteRequests/default",
+        "name": "default",
+        "type": "Microsoft.DataProtection/resourceGuards/deleteResourceGuardProxyRequests"
```

Comment:
This is not the right type for the resource ID.  Is this also a real example of the response from these GETs?  What value are they providing?

Reply:
> This is not the right type for the resource ID. Is this also a real example of the response from these GETs? What value are they providing?

Hi, This Type, we are getting from GET API call response. These changes are intended. Here type indicated the type of the critical operation request is protected by the ResourceGuard.

Here, Microsoft.DataProtection is provider namespace. resourceGuards is the type of resource used to protect the critical operation. deleteResourceGuardProxyRequests is critical operation request. Here, deletion of ResourceGuardProxy is the critical operation.

Reply:
The "type" in your example is "Microsoft.DataProtection/resourceGuards/deleteResourceGuardProxyRequests" but the resource ID is of type "Microsoft.DataProtection/resourceGuards/disableSoftDeleteRequests".  Please go through all the examples and make sure they are correct.  

What does this resource tell the user?  There are no resource specific properties.  Is the existence of one of these resources what triggers the resource guard to "guard" something?  If so, why doesn't the resourceGuard itself just have properties that enable/disable these features?

Reply:
Fixed and rechecked all the example json files. 

Here Backup Admin owns the vault. And that Vault will be protected by ResourceGuard resource. ResourceGuard resource will be created under security admin. Thus when any critical operation will be performed in the vault, permission is needed from both the admin, Thus we are enforcing multi user authentication with this feature. Here one to one mapping is there between a resource (vault) and resourceGuard. This request object is needed for requestor approval  model. Details are mentioned https://microsoft.sharepoint.com/:w:/t/AzureBackupTeam/EXw8RJ8P9KRNr5f7uEwE3swB-YfMbddSrwp6J7-FRVaudA?e=pbr0kQ&CID=086C017D-8298-43B9-97E7-BFCAC023F5A3&wdLOR=cDC5E2EEE-AC6A-4E35-A61B-08A7E05E2BCC